src/CalcManager/UnitConverter.cpp

```diff
@@ -922,17 +922,25 @@ void UnitConverter::InitializeSelectedUnits()
     vector<Unit> curUnits = itr->second;
     if (!curUnits.empty())
     {
+        bool isFromRestored = find(curUnits.begin(), curUnits.end(), m_fromType) != curUnits.end();
+        bool isToRestored = find(curUnits.begin(), curUnits.end(), m_toType) != curUnits.end();
```

Comment:
NIT: rename `isFromUnitValid` and `isToUnitValid` + a comment explaining why we check them.

Reply:
small optimization: 
```
 bool isFromRestored = m_fromType == EMPTY_UNIT && find(curUnits.begin(), curUnits.end(), m_fromType) != curUnits.end();
```###

Reply:
> NIT: rename `isFromUnitValid` and `isToUnitValid` + a comment explaining why we check them (units available for this region)

Thanks, @rudyhuyn !
I updated as suggested. Can you take a look at added comments please? I am afraid that you might have misunderstood why I was checking those values.

Reply:
The comment is right but another positive aspect of this check is that you will override units when they don't exist, many reasons possible:
- a unit was removed by a previous update (or moved to another category)
- the unit restored is not available for the current locale (user selected Pyeong, then changed region and started again the app)


-

Reply:
Ah,, makes sense!
As for the second case, it shouldn't happen once #444 lands since the PR won't allow restoring values that aren't valid for the current locale in the first place.